source/installation/install.rst

```diff
@@ -19,7 +19,7 @@ Download the last release of TractoFlow pipeline:
 
 ::
 
-    $> wget https://github.com/scilus/tractoflow/releases/download/2.2.1/tractoflow-2.2.1.zip && unzip tractoflow-2.2.1.zip
+    $> wget https://github.com/scilus/tractoflow/releases/download/2.3.0/tractoflow-2.3.0.zip && unzip tractoflow-2.3.0.zip
```

Comment:
If you say to use -r in the launch page, you need to install with the nextflow pull

Reply:
Fixed